build.sbt

```diff
@@ -1,10 +1,10 @@
 import ReleaseTransformations._
 import sbtversionpolicy.withsbtrelease.ReleaseVersion
 
-val contentEntityVersion = "3.0.3"
-val contentAtomVersion = "4.0.4"
+val contentEntityVersion = "4.0.0"
+val contentAtomVersion = "6.0.0"
```

Comment:
Currently we can't update `com.gu:content-atom-model-thrift` tp `6.0.0` because `com.gu:content-api-models-scala` depends on `4.0.4`

```
[error] 	* com.gu:content-atom-model-thrift:6.0.0 (early-semver) is selected over 4.0.4
[error] 	    +- com.gu:apps-rendering-api-models_2.13:10.0.1-SNAPSHOT (depends on 6.0.0)
[error] 	    +- com.gu:content-api-models-scala_2.13:26.0.0        (depends on 4.0.4)
```